Lib/test/test_capi/test_long.py

```diff
@@ -425,6 +425,34 @@ def test_long_asvoidptr(self):
         self.assertRaises(OverflowError, asvoidptr, -2**1000)
         # CRASHES asvoidptr(NULL)
 
+    def _test_long_aspid(self, aspid):
+        # Test PyLong_AsPid()
+        from _testcapi import SIZEOF_PID_T
+        bits = 8*SIZEOF_PID_T
+        PID_T_MIN = -2**(bits-1)
+        PID_T_MAX = 2**(bits-1)-1
```

Comment:
PEP 8 nitpicking :-)

```suggestion
        bits = 8* SIZEOF_PID_T
        PID_T_MIN = -2 ** (bits - 1)
        PID_T_MAX = 2 ** (bits - 1) - 1
```

Reply:
```rst
- If operators with different priorities are used, consider adding
  whitespace around the operators with the lowest priority(ies). Use
  your own judgment; however, never use more than one space, and
  always have the same amount of whitespace on both sides of a binary
  operator::

      # Correct:
      i = i + 1
      submitted += 1
      x = x*2 - 1
      hypot2 = x*x + y*y
      c = (a+b) * (a-b)

  ::

      # Wrong:
      i=i+1
      submitted +=1
      x = x * 2 - 1
      hypot2 = x * x + y * y
      c = (a + b) * (a - b)
```

Reply:
It's just that proposed format is more readable to me, you can ignore my suggestion.

Apparently, my assumption about PEP 8 formatting of + and * operators were wrong.